python/ray/train/v2/_internal/state/export.py

```diff
@@ -77,10 +80,8 @@ def train_run_attempt_to_proto(attempt: TrainRunAttempt) -> ProtoTrainRunAttempt
         attempt_id=attempt.attempt_id,
         status=_RUN_ATTEMPT_STATUS_MAP[attempt.status],
         status_detail=attempt.status_detail,
-        start_time=_ms_to_ns(attempt.start_time_ms),
-        end_time=_ms_to_ns(attempt.end_time_ms)
-        if attempt.end_time_ms is not None
-        else None,
+        start_time_ns=_ms_to_ns(attempt.start_time_ms),
+        end_time_ns=_ms_to_ns(attempt.end_time_ms),
```

Comment:
why's this thing saved as ms to begin with?

Reply:
That is a good question, I think it was done this way in the V1 schema. 

I can update the python schema to also use nanonseconds, and replace the following logic:

https://github.com/ray-project/ray/blob/79aa176e92ce71aff3d076cf8aafec9bbbea1425/python/ray/train/v2/_internal/state/state_manager.py#L23-L24